docs/source/aws.rst

```diff
@@ -96,6 +96,16 @@ AWS Security Groups can be provisioned using this resource.
    action. This is because other resources may depend on the now existing
    resource.
 
+aws_ec2_vpc_net
+---------------
+
+AWS VPC networks can be provisioned using this resource.
+
+* :docs1.5:`Topology Example <workspace/topologies/aws-ec2-vpc-net.yml>`
```

Comment:
should this be `workspaces/aws/topologies/aws-ec2-vpc-net.yml`?

Reply:
aws folder doesnot have a topologies folder however, the above file is referenced as example . For tests workspaces/aws/Pinfile is being used

Reply:
@samvarankashyap I don't understand your reply to @14rcole 

Where in this commit is there a file workspace/topologies/aws-ec2-vpc-net.yml?

Reply:
@p3ck Its in docs/source/examples/workspaces/aws/topologies/aws-ec2-vpc-net.yml 
my bad @14rcole was correct . I need to modify the path in docs.